py_zipkin/zipkin.py

```diff
@@ -542,6 +543,26 @@ def update_binary_annotations(self, extra_annotations):
             # the binary annotations for the logging context directly.
             self.logging_context.tags.update(extra_annotations)
 
+    def add_annotation(self, annotation_val, timestamp=None):
```

Comment:
nit: I'd rename `annotation_val` to just `value`, not a big deal though

Reply:
That makes sense!  I just renamed that parameter and pushed.